src/js/login/login.mjs

```diff
@@ -1,4 +1,5 @@
 import { login } from "./login-handler.mjs";
+import { storageSave } from "../storage/localstorage.mjs";
```

Comment:
```suggestion
```

You are not using this import

Reply:
This file is the area where most attention is needed. It is not working as intended because login is `async` and should be awaited. Please see feedback branch where I have restructured this file slightly to make it work correctly.

Reply:
Thank you. Just implemented the changes. I am new to feedback branch, so I committed the changes in js2 in the way I know how.

Reply:
the storageSave import is there in case I need to store the username from login for use in profile page functions. I don't know If I even have time for that, but commented it out for now.